README.md

```diff
@@ -3,6 +3,8 @@
 [![Build Status](https://travis-ci.org/cuspaceflight/firefish.svg?branch=master)](https://travis-ci.org/cuspaceflight/firefish)
 [![Coverage
 Status](https://coveralls.io/repos/cuspaceflight/firefish/badge.svg?branch=master&service=github)](https://coveralls.io/github/cuspaceflight/firefish?branch=master)
+[![Documentation
```

Comment:
Yeah - I accidentally managed to pull Boyuan's stlJoin branch into my master whilst playing around. Wasn't sure how to undo it and so deleted everything and started again. It's probably an artifact from that I guess.